Makefile

```diff
@@ -1,73 +1,66 @@
-WORKDIR      := $(shell pwd)
-NATIVEOS	 := $(shell go version | awk -F '[ /]' '{print $$4}')
-NATIVEARCH	 := $(shell go version | awk -F '[ /]' '{print $$5}')
-INTEGRATION  := postgresql
-BINARY_NAME   = nri-$(INTEGRATION)
-GO_PKGS      := $(shell go list ./... | grep -v "/vendor/")
-GO_FILES     := ./src/
-GOTOOLS       = github.com/kardianos/govendor \
-		gopkg.in/alecthomas/gometalinter.v2 \
-		github.com/axw/gocov/gocov \
-		github.com/AlekSi/gocov-xml \
+export PATH := $(PATH):$(GOPATH)/bin
+
+INTEGRATION     := postgresql
+
+BINARY_NAME      = nri-$(INTEGRATION)
+INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
+CONFIG_DIR       = /etc/newrelic-infra/integrations.d
+GO_FILES        := ./src/
+GOFLAGS			 = -mod=mod
+GOLANGCI_LINT	 = github.com/golangci/golangci-lint/cmd/golangci-lint
+GOCOV            = github.com/axw/gocov/gocov
+GOCOV_XML		 = github.com/AlekSi/gocov-xml
 
 all: build
 
-build: check-version clean validate test compile
+build: clean validate test compile
+
+build-container:
+	docker build -t nri-postgresql .
 
 clean:
 	@echo "=== $(INTEGRATION) === [ clean ]: Removing binaries and coverage file..."
 	@rm -rfv bin coverage.xml
 
-tools: check-version
-	@echo "=== $(INTEGRATION) === [ tools ]: Installing tools required by the project..."
-	@go get $(GOTOOLS)
-	@gometalinter.v2 --install
 
-tools-update: check-version
-	@echo "=== $(INTEGRATION) === [ tools-update ]: Updating tools required by the project..."
-	@go get -u $(GOTOOLS)
-	@gometalinter.v2 --install
+format:
+	sh scripts/format.sh
```

Comment:
isn't the linter checking that the code has been correctly formatted?

Reply:
I guess that it could depend on the linter configuration.  For example, the linter won't fail if there are empty lines, but the format would fix it